legacy/PimExportHelper.js

```diff
@@ -43,15 +43,21 @@ class PimExportHelper {
    * @param {string} fieldApi
    */
   getValue(object, fieldApi) {
+    if (!object || !fieldApi) {
+      return null
+    }
     let fields = fieldApi.split('.');
     let queryResult = object;
-    fields.forEach(field => {
+    for (const field of fields) {
+      if (!queryResult) {
```

Comment:
not sure this is needed, you already return null if `object` is null

Reply:
So i can use `break` to prevent null pointer

Reply:
but the variable `queryResult` doesn't change when you iterate through the array. Unless I am missing something.

Reply:
never mind on that one as well. I saw that queryResult does change later in the logic.